backend/src/middleware/auth_middleware.rs

```diff
@@ -87,7 +87,7 @@ where
             let fut = self.service.call(req);
             return Box::pin(async move {
                 let res = fut.await?;
-                Ok(res.map_body(|_, body| AnyBody::from_message(body)))
+                Ok(res.map_body(|_, body| AnyBody::new_boxed(body)))
```

Comment:
It's weird, I can't find the definition of new_boxed.

Reply:
https://docs.rs/actix-http/3.0.0-beta.13/src/actix_http/body/body.rs.html#51
https://docs.rs/actix-http/3.0.0-beta.13/src/actix_http/body/body.rs.html#236

Reply:
Got it. I think the version of actix-web should be updated in the cargo.toml. 

Current:
actix-web = "4.0.0-beta.11"
actix-http = "3.0.0-beta.12"

It seems like the beta version will be updated automatically if the Cargo.lock does not exist.